### DISCUSSION TITLE
status of outputs

### DISCUSSION THREAD

**Discussion by Papablaubaer** [arendst/Tasmota#15597] [Q&A] [answered]
I would like to send the status of the outputs cyclically per rule. How can I query the status of an output in a rule without the output being changed?

**Comment by sfromis** [comment, marked as answer]
Depends on what you mean by "outputs". If you means relays (or similar), the `Power0` command publishes status to MQTT. Anyway, the status of these outputs is *already* published every TelePeriod, default 5 minutes (unless you change it). If you want to "send" by other means, you can create rules picking up each power output status, and do whatever.

**Comment by Papablaubaer** [comment]
I would like something like this:
ON Time#Minute DO Websend [myserver.com] /fancy/%power1%
Where %Power1% stands for the status (1 or 0) of relay1 or dimmer1

**Reply by sfromis** [reply]
Well, in that case first read the rules syntax, as you have no such variables.

`on tele-Power1 do WebQuery http://myserver.com/fancy/%value% GET endon` could work if you have a TelePeriod of 1 minute. Me using WebQuery in the example was just because this syntax (in recent versions) is "less weird".

Of course, if you have your backend subscribe to MQTT messages instead, things are much simpler.

**Reply by Papablaubaer** [reply]
Great thanks that works very well
